addons/mass_mailing/wizard/mailing_mailing_schedule_date_views.xml

```diff
@@ -19,7 +19,7 @@
     </record>
 
     <record id="mailing_mailing_schedule_date_action" model="ir.actions.act_window">
-        <field name="name">When do you want to send your mailing?</field>
+        <field name="name">When should our postal pigeons take flight?</field>
```

Comment:
huh? not sure we are talking about a mass mailing, but if that is the case, I would be confused seeing that message

Reply:
I think we should replace "postal pigeons" by "messenger pigeons" as it is the correct form.